notebooks/257-llava-multimodal-chatbot/257-videollava-multimodal-chatbot.ipynb

```diff
@@ -81,7 +81,7 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %pip install -q "transformers&gt;=4.31.0,&lt;4.35.0" einops peft opencv_python decord pytorchvideo sentencepiece protobuf "openvino&gt;=2023.2.0" "nncf&gt;=2.7.0" gradio
</pre><p>is it still required to have upperbound for transformers here?</p><p>LLaVA itself  migrated on transformers &gt;=4.37, maybe here is the same?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
For `transformers>=4.37` VideoLLaVA repository imports raise the following error:
```
File [~/openvino_notebooks/notebooks/257-llava-multimodal-chatbot/Video-LLaVA/videollava/model/multimodal_encoder/languagebind/image/modeling_image.py:11](http://10.75.201.252:8888/lab/workspaces/auto-Y/tree/257-llava-multimodal-chatbot/257-llava-multimodal-chatbot/Video-LLaVA/videollava/model/multimodal_encoder/languagebind/image/modeling_image.py#line=10)
      9 from transformers import PreTrainedModel, add_start_docstrings
     10 from transformers.modeling_outputs import BaseModelOutput, BaseModelOutputWithPooling
---> 11 from transformers.models.clip.modeling_clip import CLIPMLP, CLIPAttention, CLIPTextEmbeddings, CLIPVisionEmbeddings, \
     12     CLIPVisionModelWithProjection, CLIPTextModelWithProjection, _expand_mask, CLIPOutput, clip_loss
     13 from transformers.utils import add_start_docstrings_to_model_forward, replace_return_docstrings
     15 from .configuration_image import LanguageBindImageConfig, CLIPVisionConfig, CLIPTextConfig

ImportError: cannot import name '_expand_mask' from 'transformers.models.clip.modeling_clip' (/home/gta/.local/lib/python3.10/site-packages/transformers/models/clip/modeling_clip.py)
```